### DISCUSSION TITLE
Issue with Opening Links in Freeplane 1.11.13

### DISCUSSION THREAD

**Discussion by NivedithaVeerapuraSiddappa** [freeplane/freeplane#1873] [Help]
Hello,

Experiencing an issue where links to folders in Freeplane mind maps created with version 1.11.3 are not opening via File Explorer after updating Freeplane to 1.11.13.

Here are the details:
Mind map created with version: 1.11.3
Freeplane Current Version: 1.11.13
Operating System: Win 10

I ensured that the links are correctly formatted as \\path\to\folder.
Renamed the link in the format file://C:/path/to/your/folder and link will work but has many links and all cannot be renamed. 
We have verified settings in Freeplane preferences. However, the issue persists.

Renaming 

Could you please provide guidance on resolving this issue?

Thank you for your assistance.

**Comment by macmarrum** [comment]
Hi @NivedithaVeerapuraSiddappa 

Could you please attach a mind map created with version 1.11.13 that has links which fail when opened in version 1.11.14? 
You'll need to ZIP the .mm file, otherwise GitHub won't allow you to attach it to a post.

**Reply by NivedithaVeerapuraSiddappa** [reply]
Hi @macmarrum ,
Unfortunately, the mind map contains links to confidential information and cannot be shared according to organization policies.
Receiving popup as attached when click on the links in mindmap, we cannot open the Microsoft store as in the popup to search for other apps since its blocked in the organization.
[FreeplanePopup.zip](https://github.com/user-attachments/files/15896003/FreeplanePopup.zip)

Please let me know if more information required to support us in fixing the reported concern.

**Reply by macmarrum** [reply]
Hi @NivedithaVeerapuraSiddappa 

Thanks for the information. Unfortunately the pop-up alone bring little new information; without details about the links that fail to open, there's little I can do.

Maybe you could create and share a new mind map with just the links that fail to open, instead of sharing the real mind map?
